src/test/groovy/com/rackspace/feeds/feedscatalog/feedscatalogResolveTest.groovy

```diff
@@ -8,22 +8,46 @@ import javax.xml.transform.stream.StreamSource
 
 class feedscatalogResolveTest extends BaseTest {
 
-    @Shared xslt = "./src/main/xsl/feedscatalog-resolve.xsl"
+    @Shared xslt = "./src/main/resources/feedscatalog-resolve-host.xsl"
     @Shared String allfeeds_XmlString = new File('./src/test/resources/test_allfeeds.xml').text
     @Shared String allfeeds_observer_XmlString = new File('./src/test/resources/test_allfeeds_observer.xml').text
 
+    @Shared String TENANT_ID= "1234567"
+    @Shared String NAST_ID= "MossoID_aaa1-bbb2-ccc3-ddd4-eee5"
+
     def setupMoreSpec() {
         System.setProperty("javax.xml.transform.TransformerFactory",
-                            "net.sf.saxon.TransformerFactoryImpl");
+                "net.sf.saxon.TransformerFactoryImpl");
+    }
+
+    def "Generate all feeds catalog for internal node"() {
```

Comment:
My previous comment was hidden when you updated the test. In case you haven't seen it, here it is:

It is usually good to state what is the expected behavior of the test in the title. For example, this one could be "Should change all urls to internalVip, when x-external-loc is not present"